src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -963,6 +969,12 @@ namespace HVACUnitarySystem {
 		UnitarySystem( UnitarySysNum ).DehumidInducedHeatingDemandRate = 0.0;
 		UnitarySystem( UnitarySysNum ).CoolCoilWaterFlowRatio = 0.0;
 		UnitarySystem( UnitarySysNum ).HeatCoilWaterFlowRatio = 0.0;
+		if( UnitarySystem( UnitarySysNum ).CoolCoilFluidInletNode > 0 ) Node( UnitarySystem( UnitarySysNum ).CoolCoilFluidInletNode ).MassFlowRate = 0.0;
```

Comment:
Interesting, if I delete these I don't get the control I had before. Also, we are not supposed to use Node().MassFlowRate on left side of equation so I'll have to switch to using SetComponentFlowRate.